launcher/tasks/Task.h

```diff
@@ -225,6 +225,19 @@ class TaskV2 : public QObject, public QRunnable {
         return false;
     }
 
+    void propateTotalChanged(TaskV2* job, [[maybe_unused]] double, double delta)
+    {
+        setProgressTotal(m_progressTotal + delta * job->weight());
+    }
+    void propateProcessedChanged(TaskV2* job, [[maybe_unused]] double, double delta) { setProgress(m_progress + delta * job->weight()); }
+    void propateState(TaskV2* job)
+    {
+        m_status = job->status();
+        m_details = job->details();
+        m_title = job->title();
+        emit stateChanged(this);
+    }
```

Comment:
This would only be valid for a sequential task not a concurrent one which would flicker between titles/status/details as each part proagates.

Reply:
You can already see that here: https://github.com/PrismLauncher/PrismLauncher/pull/2630/files#diff-d72243a0ccee6f23ea3894bff0ab990fb18775944eb2d9f8762a260ea18e12adR120-R128

Reply:
Ok, that makes sense. we should probably document that in the header so it's not assumed that the signal/slot needs to be connected.